src/models/AmountMoney.ts

```diff
@@ -0,0 +1,7 @@
+interface AmountMoney {
+    amount?: number;
```

Comment:
Should `amount` and `currencyCode` be required?

Reply:
- [ ] check if amount  and currencyCode are required

Reply:
Corrected. CurrencyCode is not mandatory as stated below from the sample:
amountMoney: {
        amount: 100,
        currencyCode: 'USD', // optional, use authorized location's currency code by default
      },